cypress/e2e/errors-jump-around.cy.js

```diff
@@ -1,10 +1,9 @@
 import './base.cy'
 
 describe('Errors when user skips and jumps pages', () => {
-  it('Throws a 404 when user skips to success page', () => {
 
+  it('Throws a 404 when user skips to success page', () => {
```

Comment:
Why do we give 404 instead of 400?

Reply:
`/success/` is not a valid URL anymore, hence the 404. But you're correct that a better test would be the user jumping to `/success/<random reference/` in which case should should indeed get a 400. This behaviour isn't implemented yet but will be after PR #242 is merge. So that PR should probably update this test.